packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -16,7 +16,7 @@ import 'debug.dart';
 /// During painting, the render tree generates a tree of composited layers that
 /// are uploaded into the engine and displayed by the compositor. This class is
 /// the base class for all composited layers.
-abstract class Layer {
+abstract class Layer extends TreeDiagnostics {
```

Comment:
Probably.  I think one of those has a different (less dense) tree, but maybe we should make them all the same.